http/query.go

```diff
@@ -93,14 +86,10 @@ func (r QueryRequest) Validate() error {
 		return errors.New(`request body requires either query or AST`)
 	}
 
-	if r.Type != "flux" && r.Type != "influxql" {
```

Comment:
@jsternberg This needs a docs change right?

Reply:
Is that the openapi specification?

Reply:
It looks like the openapi specification has had the influxql portions removed either always or for some time. It's not in there so this isn't listed as a valid option according to our api contract.